pkg/apis/app/v1alpha1/kogitoapp_types.go

```diff
@@ -132,6 +132,20 @@ type KogitoAppBuildObject struct {
 	MavenMirrorURL string `json:"mavenMirrorURL,omitempty"`
 }
 
+// AddEnvironmentVariable adds new environment variable to build environment variables
+func (k *KogitoAppBuildObject) AddEnvironmentVariable(name, value string) {
+	if k == nil {
```

Comment:
Does it make sense this check? I mean using this method by:

```go
k.AddEnvironmentVariable where k is null
```

Reply:
Good question, I have put it there as it is done i other methods.
Will try if it is usable.

Reply:
Actually, the nil check has sense. For example this is a "valid" code:
```
	var spec *KogitoServiceSpec
	spec.AddEnvironmentVariable("qq", "ww")
```
Function `AddEnvironmentVariable` is called with `spec = nil`